Functions.Templates/Templates/CosmosDBTrigger-Python/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "defaultFunctionName": "CosmosTriggerPython",
+  "description": "$CosmosDBTrigger_description",
+  "name": "Cosmos DB trigger",
+  "language": "JavaScript",
+  "category": [
+    "$temp_category_core",
+    "$temp_category_dataProcessing"
+  ],
+  "categoryStyle": "cosmosDB",
+  "enabledInTryMode": false,
+  "userPrompt": [
+    "connectionStringSetting",
+    "databaseName",
+    "collectionName",
+    "leaseCollectionName",
+    "createLeaseCollectionIfNotExists"
+  ],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.CosmosDB",
+      "version": "3.0.0-beta7"
+    }
+  ]

```

Comment:
Also need to add filters property for vs code. like this template 
Sample
https://github.com/Azure/azure-functions-templates/blob/v2.x/Functions.Templates/Templates/BlobTrigger-Python/metadata.json
```
    "filters": [
        "Python3"
    ]
```

Reply:
Done